core/src/main/java/org/apache/iceberg/deletes/PositionDeleteWriter.java

```diff
@@ -107,4 +117,13 @@ public DeleteFile toDeleteFile() {
   public DeleteWriteResult result() {
     return new DeleteWriteResult(toDeleteFile(), referencedDataFiles());
   }
+
+  private Metrics metrics() {
+    Metrics metrics = appender.metrics();
+    if (referencedDataFiles.size() > 1) {
+      return MetricsUtil.copyWithoutFieldBounds(metrics, SINGLE_REFERENCED_FILE_BOUNDS_ONLY);
```

Comment:
Looks like the idea is only to drop the bounds for `_file` and `_pos`? I guess that would mean that we keep data column ranges that might be used for filtering?

Reply:
I am not aware of any engines that persist deleted rows but I opted in for an incremental change in behavior.